core/src/test/java/com/sequenceiq/cloudbreak/util/StackUtilTest.java

```diff
@@ -166,11 +167,11 @@ void collectAndCheckReachableNodes() throws NodesUnreachableException {
         Set<Node> nodes = new HashSet<>();
         nodes.add(new Node("1.1.1.1", "1.1.1.1", "1", "m5.xlarge", "node1.example.com", "worker"));
         nodes.add(new Node("1.1.1.3", "1.1.1.3", "3", "m5.xlarge", "node3.example.com", "worker"));
-        when(hostOrchestrator.getResponsiveNodes(nodesCaptor.capture(), any())).thenReturn(new NodeReachabilityResult(nodes, Set.of()));
+        when(hostOrchestrator.getResponsiveNodes(nodesCaptor.capture(), any(), anyBoolean())).thenReturn(new NodeReachabilityResult(nodes, Set.of()));
```

Comment:
why anyBoolean() instead of a definitive value? this approach suggests me that the verification doesn't aware of which path got tested

Reply:
sam for the other cases for `getResponsiveNodes`

Reply:
as discussed, I will fix these in another PR

Reply:
as agreed with @horadla23, we're going to address this in a different ticket